src/main/java/de/uni_passau/fim/se2/pipeline_helper/Main.java

```diff
@@ -113,10 +113,19 @@ public Integer call() throws Exception {
             validateParams();
 
             final Checker checker = buildChecker();
-            final CheckerResult result = checker.check();
-            CheckerResultWriter.writeFeedback(parent.outputDirectory, result);
+            CheckerResult result;
```

Comment:
```suggestion
            final CheckerResult result;
```

Reply:
IntelliJ complains here:
"Variable 'result' might already have been assigned to"

Reply:
okay